app/javascript/components/menus/index/menus-table.vue

```diff
@@ -246,7 +246,12 @@ export default {
       return returnArray;
     },
     lastMenu(idx) {
-      return (idx === this.menus.length - 1);
+      const finalRowsCount = 2;
+      if (idx >= this.menus.length - finalRowsCount) {
+        return true;
+      }
+
+      return false;
```

Comment:
acá podi retornar directamente la condición
```suggestion
      const finalRowsCount = 2;
      return idx >= this.menus.length - finalRowsCount;
```

Reply:
wenisimo, voy